src/Microsoft.AspNet.Hosting.WindowsService/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNet.Hosting.Internal;
+using Microsoft.Extensions.Configuration;
+using System.ServiceProcess;
+
+namespace Microsoft.AspNet.Hosting.WindowsService
+{
+    public class Program : ServiceBase
+    {
```

Comment:
Not sure this should be called program. It's somethings users should derive from in their apps right?